ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/s3_utilities.py

```diff
@@ -40,113 +50,151 @@
     'external_codes.csv'
     ]
 
-# Mitre shell script files that need to be downloaded; used to generated synthetic data.
+'''
+Mitre shell script files that need to be downloaded; used to generated synthetic data.
+'''
 code_script_files = [
     'national_bfd.sh',
     'national_bfd_v2.sh'
     ]
 
-# Function to download proprietary Mitre mapping files from an S3 bucket; the files
-# are necessary to generate synthetic data. The mapping filenames are listed in an
-# array of filenames so this function loops over names in the code_map_files array.
-#
-# Param: target_dir : unix filesystem directory where downloaded S3 file(s) are written to.
-# Raises a python exception if failure to download file.
+'''
+Function to download proprietary Mitre mapping files from an S3 bucket; the files
+are necessary to generate synthetic data. The mapping filenames are listed in an
+array of filenames so this function loops over names in the code_map_files array.
+
+Param: target_dir : unix filesystem directory where downloaded S3 file(s) are written to.
+
+Raises a python exception if failure to download file.
+'''
```

Comment:
And this would technically go inside right under the method def (Which I don't love tbh, but i've been following for consistency)